features/messageevent.yml

```diff
@@ -0,0 +1,17 @@
+name: MessageEvent
+description: "The MessageEvent interface represents a message received by a target object."
+spec: https://html.spec.whatwg.org/multipage/comms.html#the-messageevent-interface
+group: messaging
+compat_features:
+  # Already generated in Broadcast channel.
+  # Revisit once the "keys in impl or feature" question is answered.
+  #- api.BroadcastChannel.message_event
+  - api.DedicatedWorkerGlobalScope.message_event
+  - api.MessageEvent
+  - api.MessageEvent.MessageEvent
+  - api.MessageEvent.data
+  - api.MessageEvent.lastEventId
+  - api.MessageEvent.origin
+  - api.MessageEvent.source
+  - api.Window.message_event
+  - api.Worker.message_event
```

Comment:
Continuing from https://github.com/web-platform-dx/web-features/pull/1913#discussion_r1792714432.

OK, thank you for bringing this up at our standup.

> `MessageEvent` is implemented in numerous places, but doesn't seem like a standalone feature... but `messageerror` was already done, so I followed that precedent here, adding the various implementation keys as well.

I think it's a bad precedent and we shouldn't honor it. 😅 

Truthfully, I think there's a little something to `messageerror` being a feature, but it's definitely a transitional feature. Someday, those keys ought to fold into their parent interfaces and they'll be an unremarked upon convention around message-sending APIs. This is where fixing https://github.com/web-platform-dx/web-features/issues/1173 would be helpful.

I think the actual task here is to move the keys to `postmessage.yml`.